okhttp-tests/src/test/java/com/squareup/okhttp/TestUtil.java

```diff
@@ -15,4 +16,8 @@ private TestUtil() {
     }
     return result;
   }
+
+  public static boolean isEqualCollections(Collection<?> a, Collection<?> b) {
```

Comment:
Or even better, just change matchingPins to return a `Set<ByteString>` instead of a `List<ByteString>`. The cost difference is negligible, and semantically a set is more correct. (you don't have duplicates; you don't care about order).

(Please use `LinkedHashSet`; I'm addicted to determinism)